src/template.test.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-env jest */
+
+const nunjucks = require('nunjucks')
+const configPaths = require('../config/paths.json')
+
+describe('Template', () => {
+  describe('with default nunjucks configuration', () => {
+    it('should not have any whitespace before the doctype', () => {
+      nunjucks.configure(configPaths.src)
+      const output = nunjucks.render('./template.njk')
+      expect(output.charAt(0)).toEqual('<')
+    })
+  })
+  describe('with nunucks block trimming enabled', () => {
+    it('should not have any whitespace before the doctype', () => {
+      nunjucks.configure(configPaths.src, {
+        trimBlocks: true,
+        lstripBlocks: true
+      })
+      const output = nunjucks.render('./template.njk')
+      expect(output.charAt(0)).toEqual('<')
```

Comment:
could we do `output.startsWith('<!DOCTYPE html>')`?

Reply:
I did wonder whether to check for more, I just thought that as long as it stars with an angle bracket then we know it has no leading whitespace. I didn't want to tie the test to a specific doctype as we don't really care here?

Reply:
Good enough for me 👍